added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/profiles/FetchProfiles.md

title: Fetch Profiles
displayed_sidebar: developSidebar
---

<head>

>   <title>Fetch Profiles on the Subsocial Blockchain | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>How To Profiles With The Subsocial JS SDK</title>
```
